### PR TITLE
[UPDATED]Cards

### PR DESCRIPTION
**Pull request by ApoorvDubey23**
changed the styling of cards when hovered and also the width of cards, also center aligned the cards components

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "astro-icon": "^0.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "sass": "^1.69.4",
```
Comment (EshaanAgg) [inline package.json:23]: Why have you added SASS as a dependency?

**Review by EshaanAgg — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ApoorvDubey23) [reply package.json:23]: to add the hover effect of View in Detail button
As it was a component I copied from internet which required SASS

Comment (EshaanAgg) [reply package.json:23]: Please use a SASS to CSS transpiler to convert it to CSS and then add those styles. Adding SASS for just one component is not worth the pain in building.

Comment (ApoorvDubey23) [reply package.json]: please check

File: package-lock.json
Comment (EshaanAgg) [inline package-lock.json:1]: Revert the changes in this file as well.

**Review by EshaanAgg — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (ApoorvDubey23) [reply package-lock.json:1]: done, the change left is the change in position of last closing curly bracket.

**Review by EshaanAgg — APPROVED** [review]
